README.md

```diff
@@ -1,29 +1,41 @@
 # GroupedUp
 
 This is the repository for group 47 in tutorial 0201. 
-The purpose of our application is too create an app for UofT students to find groups! There are 8 features: login/logout, create group, edit your user or group profile, Apply/Leave/Cancel/Approve group application, and find groups through our Matching Algorithm.
+The purpose of our application is too create an app for UofT students to find groups! There are 8 features: login/logout, create group, edit user or group profile, Apply/Leave/Cancel/Approve group application, and find groups through our Matching Algorithm.
```

Comment:
> purpose of our application is too create

should be **to**, not too

also, is Matching Algorithm capitalized? someone pitch in.

Reply:
Thanks for catching that, just updated it